src/main/resources/avro/assaymetadata.avdl

```diff
@@ -0,0 +1,146 @@
+@namespace("org.ga4gh.models")
+
+/**
+This protocol defines metadata objects describing experimental or analytical procedures, performed on biological objects (e.g. biosamples) or readouts from experimental procedures.
+*/
+
+protocol Assaymetadata {
+
+  import idl "common.avdl";
+  import idl "metadata.avdl";
+
+  /**
+  An analysis contains an interpretation of one or several experiments.
+  (e.g. SNVs, copy number variations, methylation status) together with
+  information about the methodology used.
+  */
+  record Analysis {
+
+    /**
+    Formats of id | name | description | accessions are described in the
+    documentation on general attributes and formats.
+    */
+    string id;
+
+    union { null, string } name = null;
+
+    union { null, string } description = null;
+
+    /**
+    The time at which this record was created.
+    Format: :ref:`ISO 8601 <metadata_date_time>`
+    */
+    union { null, string } createDateTime = null;
```

Comment:
changed the field name, but not the type to makes existing objects.  Adding  *DataTime didn't really make the meaning clearer.  That is `created'